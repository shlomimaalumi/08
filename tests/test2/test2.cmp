|RAM[5002|RAM[5004|RAM[5006|RAM[5008|RAM[5010|
|      2 |      5 |     13 |     34 |     89 |
