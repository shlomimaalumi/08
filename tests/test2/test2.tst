load test2.asm,
output-file test2.out,
compare-to test2.cmp,
output-list RAM[5002]%D1.6.1 RAM[5004]%D1.6.1 RAM[5006]%D1.6.1 RAM[5008]%D1.6.1 RAM[5010]%D1.6.1;

repeat 1000000 {
  ticktock;
}

output;
