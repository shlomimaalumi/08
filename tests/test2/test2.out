|RAM[5002|RAM[5004|RAM[5006|RAM[5008|RAM[5010|
